File: easy/3783_mirror_distance_of_an_integer/solution.py
class Solution:
    def mirrorDistance(self, n: int) -> int:
        n_copy = n
        reversed_n = 0

        while n_copy:
            reversed_n = reversed_n * 10 + n_copy % 10
            n_copy //= 10

        return abs(n - reversed_n)

File: easy/3783_mirror_distance_of_an_integer/solution.cpp
#include <cmath>

class Solution {
public:
  int mirrorDistance(int n) {
    int nCopy = n;
    int reverseN = 0;

    while (nCopy) {
      reverseN = reverseN * 10 + nCopy % 10;
      nCopy /= 10;
    }

    return std::abs(n - reverseN);
  }
};
